source/features/clean-rich-text-editor.tsx

```diff
@@ -1,30 +1,6 @@
 import './clean-rich-text-editor.css';
-import select from 'select-dom';
-import onetime from 'onetime';
 import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-function hideButtons(): void {
-	document.body.classList.add('rgh-clean-rich-text-editor');
-}
-
-function hideTextareaTooltip(): void {
```

Comment:
This is [one of my earliest contributions](https://github.com/refined-github/refined-github/pull/2375) to RGH 2 years ago. Time flies 🚀

Reply:
No more `title` anywhere? 🎉

Reply:
It really just selects comment boxes which doesn't have `title` anymore.